lib/content-services/src/lib/category/categories-management/categories-management.component.ts

```diff
@@ -177,10 +177,18 @@ export class CategoriesManagementComponent implements OnInit, OnDestroy {
         this.cancelExistingCategoriesLoading$.complete();
     }
 
+    get isNameCategoryVisible(): boolean {
+        return (!this.categoryNameControlVisible && this.categories?.length > 0) || this.categoryNameControlVisible;
+    }
+
     get categoryNameControl(): FormControl<string> {
         return this._categoryNameControl;
     }
 
+    get isCategoryEmpty(): boolean {
```

Comment:
missing documentation comments for public api

Reply:
I have updated the documentation here https://github.com/Alfresco/alfresco-ng2-components/blob/62bdd33c10964bcb631cfe050e39c5190439926e/docs/content-services/components/categories-management.component.md
Please review this and suggest to me if anything needs to be updated

Reply:
@DenysVuika The documentation is updated. Kindly confirm whether this is done correctly or if any update is required.

Reply:
for public docs, you just need to add code comments above the api, we generate the .md files with the tool, you just need to have 

```
/*
* Returns `true` if category is empty, otherwise `false`
*/
get isCategoryEmpty(): boolean 
```